File: expresslk-services/ExpressLK-core/src/main/java/lk/express/Reservation.java
package lk.express;

import java.util.Date;

import javax.xml.bind.annotation.XmlType;

/**
 * Temporary data structure to load seat reservation data from bookingItems and
 * heldItems.
 */
@XmlType(name = "Reservation", namespace = "http://express.lk")
public class Reservation {

	private boolean isDummy = false;
	// booking item
	private int scheduleId;
	private int fromBusStopId;
	private int toBusStopId;
	// booking item passenger
	private String seatNumber;
	private Boolean journeyPerformed;
	private String name;
	private String nic;
	private String gender;
	private String mobile;
	private String email;
	private String passengerType;
	// booking
	private Integer bookingId;
	private String bookingReference;
	private String bookingRemarks;
	private Integer bookingUserId;
	private Date bookingTime;
	private String bookingStatusCode;
	private Integer bookingAgentId;
	private Double bookingChargeable;
	private Double bookingPayments;
	private String bookingPaymentModes;
	private Double bookingRefunds;

	public boolean isDummy() {
		return isDummy;
	}

	public void setDummy(boolean isDummy) {
		this.isDummy = isDummy;
	}

	public int getScheduleId() {
		return scheduleId;
	}

	public void setScheduleId(int scheduleId) {
		this.scheduleId = scheduleId;
	}

	public int getFromBusStopId() {
		return fromBusStopId;
	}

	public void setFromBusStopId(int fromBusStopId) {
		this.fromBusStopId = fromBusStopId;
	}

	public int getToBusStopId() {
		return toBusStopId;
	}

	public void setToBusStopId(int toBusStopId) {
		this.toBusStopId = toBusStopId;
	}

	public String getSeatNumber() {
		return seatNumber;
	}

	public void setSeatNumber(String seatNumber) {
		this.seatNumber = seatNumber;
	}

	public Boolean getJourneyPerformed() {
		return journeyPerformed;
	}

	public void setJourneyPerformed(Boolean journeyPerformed) {
		this.journeyPerformed = journeyPerformed;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getNic() {
		return nic;
	}

	public void setNic(String nic) {
		this.nic = nic;
	}

	public String getGender() {
		return gender;
	}

	public void setGender(String gender) {
		this.gender = gender;
	}

	public String getMobile() {
		return mobile;
	}

	public void setMobile(String mobile) {
		this.mobile = mobile;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public Integer getBookingId() {
		return bookingId;
	}

	public void setBookingId(Integer bookingId) {
		this.bookingId = bookingId;
	}

	public String getBookingReference() {
		return bookingReference;
	}

	public void setBookingReference(String bookingReference) {
		this.bookingReference = bookingReference;
	}

	public String getBookingRemarks() {
		return bookingRemarks;
	}

	public void setBookingRemarks(String bookingRemarks) {
		this.bookingRemarks = bookingRemarks;
	}

	public Integer getBookingUserId() {
		return bookingUserId;
	}

	public void setBookingUserId(Integer bookingUserId) {
		this.bookingUserId = bookingUserId;
	}

	public Date getBookingTime() {
		return bookingTime;
	}

	public void setBookingTime(Date bookingTime) {
		this.bookingTime = bookingTime;
	}

	public String getBookingStatusCode() {
		return bookingStatusCode;
	}

	public void setBookingStatusCode(String bookingStatusCode) {
		this.bookingStatusCode = bookingStatusCode;
	}

	public Integer getBookingAgentId() {
		return bookingAgentId;
	}

	public void setBookingAgentId(Integer bookingAgentId) {
		this.bookingAgentId = bookingAgentId;
	}

	public Double getBookingChargeable() {
		return bookingChargeable;
	}

	public void setBookingChargeable(Double bookingChargeable) {
		this.bookingChargeable = bookingChargeable;
	}

	public Double getBookingPayments() {
		return bookingPayments;
	}

	public void setBookingPayments(Double bookingPayments) {
		this.bookingPayments = bookingPayments;
	}

	public String getBookingPaymentModes() {
		return bookingPaymentModes;
	}

	public void setBookingPaymentModes(String bookingPaymentModes) {
		this.bookingPaymentModes = bookingPaymentModes;
	}

	public Double getBookingRefunds() {
		return bookingRefunds;
	}

	public void setBookingRefunds(Double bookingRefunds) {
		this.bookingRefunds = bookingRefunds;
	}

	public String getPassengerType() {
		return passengerType;
	}

	public void setPassengerType(String passengerType) {
		this.passengerType = passengerType;
	}
}

File: expresslk-webportal/module/Api/src/Api/Client/Soap/Core/Reservation.php
<?php

namespace Api\Client\Soap\Core;

class Reservation
{

    /**
     * @var int $bookingAgentId
     * @access public
     */
    public $bookingAgentId = null;

    /**
     * @var float $bookingChargeable
     * @access public
     */
    public $bookingChargeable = null;

    /**
     * @var int $bookingId
     * @access public
     */
    public $bookingId = null;

    /**
     * @var string $bookingPaymentModes
     * @access public
     */
    public $bookingPaymentModes = null;

    /**
     * @var float $bookingPayments
     * @access public
     */
    public $bookingPayments = null;

    /**
     * @var string $bookingReference
     * @access public
     */
    public $bookingReference = null;

    /**
     * @var float $bookingRefunds
     * @access public
     */
    public $bookingRefunds = null;

    /**
     * @var string $bookingRemarks
     * @access public
     */
    public $bookingRemarks = null;

    /**
     * @var string $bookingStatusCode
     * @access public
     */
    public $bookingStatusCode = null;

    /**
     * @var dateTime $bookingTime
     * @access public
     */
    public $bookingTime = null;

    /**
     * @var int $bookingUserId
     * @access public
     */
    public $bookingUserId = null;

    /**
     * @var boolean $dummy
     * @access public
     */
    public $dummy = null;

    /**
     * @var string $email
     * @access public
     */
    public $email = null;

    /**
     * @var int $fromBusStopId
     * @access public
     */
    public $fromBusStopId = null;

    /**
     * @var string $gender
     * @access public
     */
    public $gender = null;

    /**
     * @var boolean $journeyPerformed
     * @access public
     */
    public $journeyPerformed = null;

    /**
     * @var string $mobile
     * @access public
     */
    public $mobile = null;

    /**
     * @var string $name
     * @access public
     */
    public $name = null;

    /**
     * @var string $nic
     * @access public
     */
    public $nic = null;

    /**
     * @var int $scheduleId
     * @access public
     */
    public $scheduleId = null;

    /**
     * @var string $seatNumber
     * @access public
     */
    public $seatNumber = null;

    /**
     * @var int $toBusStopId
     * @access public
     */
    public $toBusStopId = null;
	
	
	public $passengerType = null;

    /**
     * @param int $bookingAgentId
     * @param float $bookingChargeable
     * @param int $bookingId
     * @param string $bookingPaymentModes
     * @param float $bookingPayments
     * @param string $bookingReference
     * @param float $bookingRefunds
     * @param string $bookingRemarks
     * @param string $bookingStatusCode
     * @param dateTime $bookingTime
     * @param int $bookingUserId
     * @param boolean $dummy
     * @param string $email
     * @param int $fromBusStopId
     * @param string $gender
     * @param boolean $journeyPerformed
     * @param string $mobile
     * @param string $name
     * @param string $nic
     * @param int $scheduleId
     * @param string $seatNumber
     * @param int $toBusStopId
     * @access public
     */
    public function __construct($bookingAgentId, $bookingChargeable, $bookingId, $bookingPaymentModes, $bookingPayments, $bookingReference, $bookingRefunds, $bookingRemarks, $bookingStatusCode, $bookingTime, $bookingUserId, $dummy, $email, $fromBusStopId, $gender, $journeyPerformed, $mobile, $name, $nic, $scheduleId, $seatNumber, $toBusStopId, $passengerType)
    {
      $this->bookingAgentId = $bookingAgentId;
      $this->bookingChargeable = $bookingChargeable;
      $this->bookingId = $bookingId;
      $this->bookingPaymentModes = $bookingPaymentModes;
      $this->bookingPayments = $bookingPayments;
      $this->bookingReference = $bookingReference;
      $this->bookingRefunds = $bookingRefunds;
      $this->bookingRemarks = $bookingRemarks;
      $this->bookingStatusCode = $bookingStatusCode;
      $this->bookingTime = $bookingTime;
      $this->bookingUserId = $bookingUserId;
      $this->dummy = $dummy;
      $this->email = $email;
      $this->fromBusStopId = $fromBusStopId;
      $this->gender = $gender;
      $this->journeyPerformed = $journeyPerformed;
      $this->mobile = $mobile;
      $this->name = $name;
      $this->nic = $nic;
      $this->scheduleId = $scheduleId;
      $this->seatNumber = $seatNumber;
      $this->toBusStopId = $toBusStopId;
	  $this->passengerType = $passengerType;
    }

}
